conans/client/new.py

```diff
@@ -0,0 +1,111 @@
+conanfile = """from conans import ConanFile, CMake, tools
+import os
+
+
+class {name}Conan(ConanFile):
+    name = "{name}"
+    version = "{version}"
+    license = "<Put the package license here>"
+    url = "<Package recipe repository url here, for issues about the package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {{"shared": [True, False]}}
+    default_options = "shared=False"
+    generators = "cmake"
+
+    def source(self):
```

Comment:
The issue is that with this configuration you can very easily check that you have a working system. You don't need to edit the files, add any custom clone, url... Not only for us (to test this feature), but also for users. If something fails for them (setup), they can test "conan new" + "conan export" + "conan test_package", and everything will work. Also it provides with a working example of shared/static, otherwise leaving those options there, they should be commented too.